nursery/decrypt-data-using-tea.yml

```diff
@@ -4,14 +4,43 @@ rule:
     namespace: data-manipulation/encryption/tea
     authors:
       - william.ballenthin@mandiant.com
+      - raymond.leong@mandiant.com
     scope: function
     att&ck:
       - Defense Evasion::Obfuscated Files or Information [T1027]
     mbc:
       - Defense Evasion::Obfuscated Files or Information::Encryption-Standard Algorithm [E1027.m05]
     references:
       - https://en.wikipedia.org/wiki/Tiny_Encryption_Algorithm
+    examples:
+      - 5B99FA01C72CEBC53A76CC72E9581189:0x180001027
+      - 7FD2E2E3C88675D877190ABAA3002B55:0x010004EC0
+      - 775B26D6ACCB5A4AE868983F8F5006A3:0x010018890
```

Comment:
now that we have examples, let's move this rule out of the nursery.

Reply:
in fact, all rules in this PR with examples we can move out of the nursery, awesome!